src/index.html

```diff
@@ -27,37 +32,67 @@ <h3>Community Average Step Goal</h3>
         </article>
       </article>
     </section>
+    <section class="stats">
+      
+      <article class="steps">
+        <h2>Steps</h2>
+      </article>
 
-    <section class="fitness-categories">
       <article class="water">
         <h2>Water</h2>
       </article>
-      <article class="activity">
-        <h2>Activity</h2>
-        <div class="activity-container">
-          <p class="steps">
-          </p>
-          <p class="active-minutes">
-          </p>
-          <p class="stairs">
-          </p>
-        </div>
+
+      <article class="stairs">
+        <h2>Stairs</h2>
       </article>
+
+      <article class="active-minutes">
+        <h2>Active Minutes</h2>
+      </article>
+
       <article class="sleep">
         <h2>Sleep</h2>
         <div class="sleep-container">
+          <article class="user-all-time-sleep-stats">
+            <h3>Your All-Time Sleep Stats</h3>
+          </article>
           <p class="sleep-hours">
           </p>
           <p class="sleep-quality">
           </p>
         </div>
       </article>
     </section>
+
+    <article class="date-select">
+      <label for="start">Show stats for date:</label>
+      <input type="date" id="today" name="today"
+        value="2019-09-22"
+        min="2019-06-15" max="2019-09-16">
+      <h2>Dates for Graph</h2>
+      <label for="start">Start date:</label>
+      <input type="date" id="start" name="week-start"
+        value="2019-09-16"
+        min="2019-06-15" max="2019-09-16">
+      <label for="end">End date:</label>
+      <input type="date" id="end" name="week-end"
+        value="2019-09-22"
+        min="2019-06-21" max="2019-09-22">
+    </article>
+
     <section class="graph">
       <h2 class="title-graph">Graph</h2>
+      <canvas id="stepChart" width="800" height="400"></canvas>
+      <canvas id="waterChart" width="800" height="400"></canvas>
+      <canvas id="stairsChart" width="800" height="400"></canvas>
+      <canvas id="activeMinutesChart" width="800" height="400"></canvas>
+      <canvas id="sleepChart" width="800" height="400"></canvas>
+      
     </graph>
+
```

Comment:
Since we decided to remove the Graph title and charts to just show the graphs on their own, I think we can remove the h2 class "title-graph" and solely use the "graph" class when linking to the DOM.